bank_system/settings.py

```diff
@@ -37,7 +37,9 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'banks'
+    'banks',
+    'rest_framework',
+    'rest_framework.authtoken',
 ]
```

Comment:
break down INSTALLED_APPS to 3 things
1. BUIT_IN_APPS
2. THIRD_PARTY_APPS
3. CUSTOM_APPS